recipes-ni/ni-safemode-utils/ni-safemode-utils_1.0.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "nilrt safemode utilities"
+DESCRIPTION = "nilrt distro-specific safemode utilities that provide basic system functionality."
```

Comment:
I have my reservations to all these references to "safemode". Maybe they will go away/be updated when we figure out the role of these files in a RAUC system?

Reply:
My hope is that the recipe itself will go away completely.